pages/index.tsx

```diff
@@ -17,33 +17,57 @@ import Art2 from "@components/artwork/2";
 import Art3 from "@components/artwork/3";
 
 const TOPICS = [
+  {
+    topic: "Staking Protocols",
+    artwork: (
+      <img src="/use-cases/use-case-5.png" className={styles.artworkTopic} />
+    ),
+    description: (
+      <React.Fragment>
+        Staking Protocols can substantially improve the capital efficiency in
+        the Filecoin economy - matching yield-seeking token holders and
+        liquidity-seeking storage providers. With staking protocols, token
+        holders can trustlessly pool their assets and set rules and terms by
+        which storage providers (of any size) can permissionlessly access that
+        liquidity.
+        <br />
+        <br />
+        Staking protocols can use on-chain metrics (e.g. longevity of an
+        address, history of slashing) to better assess risk while also allowing
+        storage providers of all sizes to access capital. This can help storage
+        providers fund up-front capital costs like Filecoin+ deal collateral, while
```

Comment:
```suggestion
        providers fund upfront capital costs like Filecoin Plus deal collateral, while
```

Filecoin Plus, or Fil+ AFAIK, but not Filecoin+.